ui/app/common/ui-helper/services/messagingService.js

```diff
@@ -14,7 +14,9 @@ angular.module('bahmni.common.uiHelper')
             messageObject.value = message.replace(/[\[\]]+/g, '');
             if (errorEvent) {
                 messageObject.isServerError = true;
-                this.createTimeout('error', 6000);
+                if (self.messages[level].length === 0) {
```

Comment:
Can we do `!self.messages[level].length` instead

Reply:
This has been updated